migrations/challenges/sync-all-challenges.js

```diff
@@ -0,0 +1,45 @@
+import Bluebird from 'Bluebird';
+
+import { model as Challenges } from '../../website/server/models/challenge';
+import { model as User } from '../../website/server/models/user';
+
+async function syncChallengeToMembers (challenges) {
+  challenges.forEach(async function (challenge) {
+    let users = await User.find({challenges: challenge._id}).exec();
+
+    users.forEach(async function (user) {
+      let promises = [];
+
+      promises.push(challenge.syncToUser(user));
+      promises.push(challenge.save());
+      promises.push(user.save());
+
+      await Bluebird.all(promises);
+    });
+  });
+
+  return challenges;
+}
+
+async function syncChallenges (lastChallengeDate) {
+  let query = {};
+
+  if (lastChallengeDate) {
+    query.createdOn = { $lte: lastChallengeDate };
+  }
+
+  await Challenges.find(query)

```

Comment:
`let challengesFound = await Challenges.find().limit().sort(...).exec()`

then remove the `.then` and do

```
      let syncedChallenges = await syncChallengeToMembers(challengesFound);
      let lastChallenge = challengesFound[challengesFound.length - 1];
      if (lastChallenge) await syncChallenges(lastChallenge.createdAt);
      return syncedChallenges;
```